pkg/account/delete/delete.go

```diff
@@ -60,55 +67,121 @@ func (a Account) String() string {
 	return fmt.Sprintf("%q (%s)", a.Name, a.UUID)
 }
 
-// AccountResources removes all given Resources from the given Account
+// DeleteAccountResources removes all given Resources from the given Account
 // Returns an error if any resource fails to be deleted, but attempts to delete as many resources as possible and only returns an error at the end.
-func AccountResources(ctx context.Context, account Account, resourcesToDelete Resources) error {
+func DeleteAccountResources(ctx context.Context, account Account, resourcesToDelete Resources) error {
+	totalErrorCount := deleteUsers(ctx, account, resourcesToDelete.Users) +
```

Comment:
Could the delete calls be done here in Parralel?

Reply:
I'm not sure, let's save that for a follow up